src/main/java/grocery/Grocery.java

```diff
@@ -18,10 +19,11 @@ public class Grocery {
      * @param amount Measurement of grocery.
      * @param expiration When grocery expires.
      */
-    public Grocery(String name, int amount, LocalDate expiration) {
+    public Grocery(String name, int amount, LocalDate expiration, double cost) {
```

Comment:
The addition of the cost attribute and corresponding methods (setCost and getCost) are implemented correctly. The printGrocery method has been updated to include cost information, which aligns with the feature's requirements.
Consider validating the cost in the setCost method to ensure it's a positive value, as negative costs might not be meaningful in this context.